### DISCUSSION TITLE
Incorrect key in "colour.SD_TO_XYZ_METHODS" attribute?

### DISCUSSION THREAD

**Discussion by frankkjensen** [colour-science/colour#1077] [Q&A] [answered]
I don't know if it is a bug or not, but I noticed that SD_TO_XYZ_METHODS have both 'ASTM E308' and 'astm2015' referring to the same function: sd_to_XYZ_ASTME308 at 0x0000016BAC786560

```
colour.SD_TO_XYZ_METHODS
Out[13]: CanonicalMapping({'ASTM E308': <function sd_to_XYZ_ASTME308 at 0x0000016BAC786560>, 'Integration': <function sd_to_XYZ_integration at 0x0000016BAC786440>, 'astm2015': <function sd_to_XYZ_ASTME308 at 0x0000016BAC786560>})
```

-Frank

**Comment by thomasmansencal** [comment, marked as answer]
Hey @frankkjensen,

It is an alias to `ASTM 308`: https://github.com/colour-science/colour/blob/develop/colour/colorimetry/tristimulus_values.py#L1088

Cheers,

Thomas

**Reply by frankkjensen** [reply]
Ok, thanks :-)

-Frank
